app/views/admin_request/show.html.erb

```diff
@@ -372,6 +372,11 @@
 
 <hr>
 
+<h2> Attachments </h2>
+<%= render partial: 'admin_request/some_attachments' %>
```

Comment:
Should be able to re-use the existing partial rather than duplicating it.

For example:

```erb
<%= render partial: 'admin_incoming_message/foi_attachments',
           locals: { foi_attachments: fetch_foi_attachments(@info_request) } %>
```

However, see above notes for tweaks to how we collect the attachments to render.

Reply:
This causes ugliness due to the legend in the existing partial, but if we can live with that, then that's possible. 
<img width="859" alt="Screenshot 2024-02-13 at 17 59 08" src="https://github.com/mysociety/alaveteli/assets/120410992/bd8063aa-1f22-4cc0-9210-b722e32ce5bb">